codice/ddf-ui
# Update search location drawing tools for 2D map

* Search location shapes can now be edited on the 2D map.
* A new drawing toolbar is now displayed at the top of the map.
* The 'cancel drawing' banner has been removed. The 'Draw' button now toggles to 'Cancel Drawing' when clicked.
* Fixes some bugs with how lines and polygons that cross the antimeridian are displayed.

### Testing Instructions
* Test both the basic and advanced search menus.
* Test drawing and editing all 4 shapes (polygon, bounding box, line, point-radius).
  * Drawing: draw a completely new shape.
  * Editing: edit the points/boundaries of an existing shape, and add existing points.
    * Line: add new points by clicking and dragging existing line segments, and move any points on the line.
    * Point-radius: move the center point, and change the radius by moving the circle's edge.
    * Bounding box: move any of the corners or edges.
    * Polygon: add new points by clicking and dragging existing line segments, and move any points on the line.
* Test entering shape coordinates manually, then editing the resulting shape.
* Test drawing and editing shapes that cross the antimeridian.
  * There are some limitations here, ask me for details.
* Test cancelling the drawing via the cancel button on the toolbar and the draw button on the search menu that is toggled to a cancel button when a drawing is in progress.
* Perform some basic testing on the 3D map to ensure drawing was not broken.

https://user-images.githubusercontent.com/4495447/224818911-15d8639a-900e-4a5d-992f-afba748d4276.mp4
